pkgs/development/tools/vim-vint/default.nix

```
@@ -19,8 +19,8 @@ buildPythonApplication rec {
   # Prevent setup.py from adding dependencies in run-time and insisting on specific package versions
   patchPhase = ''
```

This is better to be
```suggestion
  postPatch = ''
```